database/mock/user.mock.js

```diff
@@ -0,0 +1,89 @@
+module.exports = [
+  {
+    id: '136530bc-2d6b-4059-a978-ae49ed663156',
+    username: 'amossdale0',
+    email: 'agiacovazzo0@mashable.com',
+    password: '$2y$12$KOyzmBBn2ADOgvQ5RbNlwelqbrGcqQ45KbZYhcNU1b6ZA0dhCZGum',
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  },
+  {
+    id: '089de619-981c-45ca-adec-0d814bac03a9',
+    username: 'taddy1',
+    email: 'soffield1@boston.com',
+    password: '$2y$12$KOyzmBBn2ADOgvQ5RbNlwelqbrGcqQ45KbZYhcNU1b6ZA0dhCZGum',
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  },
+  {
+    id: '1e6b1fea-7f07-4934-aa4e-960b6560a331',
+    username: 'ldalton2',
+    email: 'khandrek2@feedburner.com',
+    password: '$2y$12$KOyzmBBn2ADOgvQ5RbNlwelqbrGcqQ45KbZYhcNU1b6ZA0dhCZGum',
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  },
+  {
+    id: '478beec0-361e-4caf-b506-7906ab763d7b',
+    username: 'acaulliere3',
+    email: 'jvanichkin3@usatoday.com',
+    password: '$2y$12$KOyzmBBn2ADOgvQ5RbNlwelqbrGcqQ45KbZYhcNU1b6ZA0dhCZGum',
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  },
+  {
+    id: 'b5e176c7-3494-4254-8920-d523529f36f0',
+    username: 'hbriance4',
+    email: 'ggoldsworthy4@wordpress.com',
+    password: '$2y$12$KOyzmBBn2ADOgvQ5RbNlwelqbrGcqQ45KbZYhcNU1b6ZA0dhCZGum',
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  },
+  {
+    id: '0505ecbb-d2d9-4230-87f6-4173620dfbc4',
+    username: 'hboundey5',
+    email: 'mbrantzen5@bbc.co.uk',
+    password: '$2y$12$KOyzmBBn2ADOgvQ5RbNlwelqbrGcqQ45KbZYhcNU1b6ZA0dhCZGum',
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  },
+  {
+    id: '844e111b-5e8f-48ca-9a8e-ceefde25479a',
+    username: 'mcadalleder6',
+    email: 'nantcliffe6@domainmarket.com',
+    password: '$2y$12$KOyzmBBn2ADOgvQ5RbNlwelqbrGcqQ45KbZYhcNU1b6ZA0dhCZGum',
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  },
+  {
+    id: 'da90dadb-df3d-4f1d-b2f9-08dd0fcb6554',
+    username: 'cdicken7',
+    email: 'eleeming7@jigsy.com',
+    password: '$2y$12$KOyzmBBn2ADOgvQ5RbNlwelqbrGcqQ45KbZYhcNU1b6ZA0dhCZGum',
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  },
+  {
+    id: 'd98f55ea-1e9d-45c7-ac61-018eac24a87c',
+    username: 'hwickenden8',
+    email: 'dvigus8@tiny.cc',
+    password: '$2y$12$KOyzmBBn2ADOgvQ5RbNlwelqbrGcqQ45KbZYhcNU1b6ZA0dhCZGum',
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  },
+  {
+    id: '32c439d7-c367-4003-872d-7abdef9dc620',
+    username: 'sashby9',
+    email: 'tjorissen9@smh.com.au',
+    password: '$2y$12$KOyzmBBn2ADOgvQ5RbNlwelqbrGcqQ45KbZYhcNU1b6ZA0dhCZGum',
+    createdAt: new Date(),
+    updatedAt: new Date(),
+  },
+  // {
+  //   username: 'Pascal Ulor',
+  //   email: 'pascal@.com.au',
+  //   password: '$2y$12$KOyzmBBn2ADOgvQ5RbNlwelqbrGcqQ45KbZYhcNU1b6ZA0dhCZGum',
+  //   createdAt: new Date(),
+  //   updatedAt: new Date(),
+  // },
```

Comment:
If there is no need for this, i think you can delete it.

Reply:
Everything else looks good. Nice work guys 🔥

Reply:
we needed to some seed data to test the database